phy/cluster/algorithms.py

```diff
@@ -155,7 +164,7 @@ def __init__(self, tempdir=None, probe=None, **kwargs):
     def _create_filter(self):
         rate = self._kwargs['sample_rate']
         low = self._kwargs['filter_low']
```

Comment:
@nippoo I had misread your comment; yes these parameters have always been in Hertz!

We can save the LFP, but in which format? We already save the HPF in temporary files (flat binary), one file per chunk (to avoid contention issues), shall we do the same for LFP too? In any case @nsteinme please add an issue otherwise we'll forget it...